.circleci/config.yml

```diff
@@ -119,7 +145,9 @@ workflows:
             branches:
               only: master
           requires:
-            - test
+            - test-core
+            - test-packages
+            - lint
             - build
             - ci/validate-commits
```

Comment:
I guess we could replace this with the tasks creating the packages? 🤔 

eg.
```yaml
requires:
  - ci/notify-success
```

Reply:
Not sure, because we don't build packages separately. Core package still is the orchestrator or the builder if you call it that imports everything and builds everything.